### PR TITLE
CA-222: Requester pays

### PR DESCRIPTION
**Pull request by helgridly**
CA-222.

Friends:
https://github.com/broadinstitute/workbench-libs/pull/228
(orch: todo)

### CODE REVIEW

**Comment by helgridly** [comment]
jenkins retest

**Comment by helgridly** [comment]
jenkins retest

**Comment by helgridly** [comment]
jenkins retest

**Comment by helgridly** [comment]
jenkins retest

File: core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala [hunk 1]
```diff
@@ -28,6 +28,7 @@ import org.joda.time.DateTime
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 import akka.http.scaladsl.model.headers.OAuth2BearerToken
 import akka.http.scaladsl.model.{StatusCodes, Uri}
+import cats.effect.IO
```
Comment (marctalbott) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala]: Is this used?

**Review by marctalbott — APPROVED** [review]

**Review by gpolumbo-broad — APPROVED** [review]

**Pull request merged**